patches/server/1044-itemstack-expose-tooltip-lines.patch

```diff
@@ -0,0 +1,30 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Yannick Lamprecht <yannicklamprecht@live.de>
+Date: Mon, 22 Jan 2024 13:27:30 +0100
+Subject: [PATCH] itemstack expose tooltip lines
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/util/CraftMagicNumbers.java b/src/main/java/org/bukkit/craftbukkit/util/CraftMagicNumbers.java
+index 509f8487a170f3dc84b091acf16df26e42391189..c0f95f569feb0ed05085e6cb6f8bacafe8d2c5a8 100644
+--- a/src/main/java/org/bukkit/craftbukkit/util/CraftMagicNumbers.java
++++ b/src/main/java/org/bukkit/craftbukkit/util/CraftMagicNumbers.java
+@@ -641,6 +641,19 @@ public final class CraftMagicNumbers implements UnsafeValues {
+         return org.bukkit.craftbukkit.CraftStatistic.getNMSStatistic(statistic).getName();
+     }
+     // Paper end
++    // Paper start - expose itemstack tooltip lines
++    @Override
++    public List< net.kyori.adventure.text.Component> tooltipLines(ItemStack itemStack, final org.bukkit.entity.Player player, final org.bukkit.inventory.TooltipOption options) {
```

Comment:
Space after `<`.
The rest of this method also seems to have some formatting issues (see missing space after switch statement, missing space after (...CraftPlayer) cast)

Reply:
Should be addressed.